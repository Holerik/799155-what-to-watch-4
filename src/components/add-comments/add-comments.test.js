// add-review.test.js
import React from 'react';
import renderer from 'react-test-renderer';
import AddComments from './add-comments.jsx';

const mockReview = {
  id: 1,
  authorId: 1,
  author: ``,
  rating: `0`,
  date: `2020-07-24T14:16:23.037Z`,
  text: ``,
};

describe(`AddComments test`, () => {
  it(`<AddComments /> should render correctly`, () => {
    const tree = renderer
      .create(<AddComments
        comments={[mockReview]}
      />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
