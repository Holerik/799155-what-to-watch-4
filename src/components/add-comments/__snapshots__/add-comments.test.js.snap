// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddComments test <AddComments /> should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        
      </cite>
      <time
        className="review__date"
        dateTime="2020-07-24T14:16:23.037Z"
      >
        July 24, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    0
  </div>
</div>
`;
